harness/tests/determined/common/experimental/test_experiment.py

```diff
@@ -229,3 +229,61 @@ def test_download_code_writes_output_to_file(
     with open(output_file, "rb") as f:
         file_content = f.read()
         assert file_content == b"b64TgzResponse"
+
+
+@mock.patch("determined.common.api.bindings.get_GetExperimentCheckpoints")
+def test_list_checkpoints_calls_bindings_with_sort_by_metric(
```

Comment:
tiny nit: someone running tests or seeing the failure won't know from the test name what the failure was about. `test_list_checkpoints_uses_bindings_sortByMetric_on_str_sort_by` does that a little better. But, like, dang that's a tiny nit, and if you prefer this then works for me (also if so, I'm interested in the reasoning so I can learn something).

Reply:
changed to `test_list_checkpoints_calls_bindings_sortByMetric_with_sort_by_str`

Reply:
nice